src/core/algorithms.js

```diff
@@ -9,4 +9,13 @@ export async function run() {
   elements
     .filter(li => li.innerText.trim().startsWith("Assert: "))
     .forEach(li => li.classList.add("assert"));
+  const css = `.assert {
```

Comment:
This is good, but we generally prefer to put the CSS into its own file. I'd suggest creating "algorithms.css" (see other modules for how to import the css using `text!`). Note that we just merged pull request #2007, which moves css into the "assets/" folder.